napari/layers/utils/color_manager.py

```diff
@@ -224,7 +224,7 @@ def _set_color(
             self.color_properties = ColorProperties(
                 name=color,
                 values=properties[color],
-                current_value=np.squeeze(current_properties[color]),
+                current_value=current_properties[color][0],
```

Comment:
I don't immediately understand this fix because I assume that current_properties should be a scalar or 1d-array like, in which case `np.squeeze` seems like the right choice.

Could you add a comment or edit the PR description to explain why this fix works?

Reply:
I agree it's a bit funny. Our the values of our `current_properties` dictionaries are 1D numpy arrays with shape (1,). Apparently when applying `np.squeeze` to a numpy array of shape (1,), it returns an array with shape (). This causes an error when trying to then map that property value to a color (our colormaps either expect a single value or a (n,) array of values).

```python
>>> import numpy as np
>>> x = np.array(['hi'])
>>> squeezed = np.squeeze(x)
>>> squeezed
array('hi', dtype='<U2')
>>> squeezed.shape
()
>>> len(squeezed)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: len() of unsized object

```

Reply:
I've added a comment to describe the fix. I am also open to other solutions!